pkg/models/monitoring/expressions/prometheus/prometheus_test.go

```diff
@@ -0,0 +1,51 @@
+package prometheus
+
+import (
+	"fmt"
+	"github.com/google/go-cmp/cmp"
+	"testing"
+)
+
+func TestLabelReplace(t *testing.T) {
+	tests := []struct {
+		expr        string
+		expected    string
+		expectedErr bool
+	}{
+		{
+			expr:        "up",
+			expected:    `up{namespace="default"}`,
+			expectedErr: false,
+		},
+		{
+			expr:        `up{namespace="random"}`,
+			expected:    `up{namespace="default"}`,
+			expectedErr: false,
+		},
+		{
+			expr:        `up{namespace="random"} + up{job="test"}`,
+			expected:    `up{namespace="default"} + up{job="test",namespace="default"}`,
+			expectedErr: false,
+		},
+		{
+			expr:        `@@@@`,
+			expectedErr: true,
+		},
+	}
+
+	for i, tt := range tests {
+		t.Run(fmt.Sprintf("%d", i), func(t *testing.T) {
+			result, err := labelReplace(tt.expr, "default")
+			if err != nil {
+				if !tt.expectedErr {
+					t.Fatal(err)
+				}
+				return
```

Comment:
return should be changed to continue?

Reply:
It is to return the Run method, and it won't break the outter loop

Reply:
I see, the return is in another function